src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -15516,23 +15456,20 @@ class FindRelatedBooksRequestPagedPageSizeOdyssey
 class GetBookAsyncFlattenedAsyncTestOnSuccessMap
 {
     // [START sample_core]
-   public static void SampleGetBook()
+   public static async Task SampleGetBookAsync()
    {
-       public static async Task SampleGetBookAsync()
-       {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
-           stt::Task<Book> response = libraryServiceClient.GetBookAsync(name);
-           // FIXME: inspect the results
-       }
-       // [END sample_core]
+       LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+       BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
+       await stt::Task<Book> response = libraryServiceClient.GetBookAsync(name);
```

Comment:
This `await` is incorrectly positioned. It should be after the `=`; and the type of `response` should not be a `Task`
`Book response = await libraryServiceClient.GetBookAsync(name);`

There are various other cases of this below.